File: algoritmos/Exercício 2/Exercicio2_UEPG/Classes/cao.js
/*Os cães possuem a ação de atacar.*/
class Cao extends Mamifero{
    constructor(peso, patas){
        super(peso, patas);
    }

    atacar(){
        let mensagem = "Sou um cão e estou atacando";
        alert(mensagem);
        return mensagem;
    }
}

File: algoritmos/Algoritmo 2/Algoritmo2/src/main/java/com/fabio/algoritmo2/Cao.java
package com.fabio.algoritmo2;

/**
 *
 * @author fabio
 */

public class Cao extends Mamifero {
    
    public Cao(double peso, int n_patas){
        super(peso, n_patas);
    }
    
    public void atacar(){
        System.out.println("estou atacando.\n");
    }
}